modules/configuring-ide.adoc

```diff
@@ -0,0 +1,199 @@
++
+The status of the workbench that you started changes from *Stopped* to *Running*. 
+. After the workbench has started, click *Open* to open the workbench notebook.
+. In the Activity Bar, click the Extensions icon. (image:images/vscode-extensions-icon.png[The Extensions icon])
+. Search for the name of the extension you want to install. 
+. Click *Install* to add the extension to your VS Code Server environment.
++ 
+The extension you installed appears in the *Browser - Installed* list on the *Extensions* panel.
+
+==== Extensions
+
+The following list describes some available third-party extensions that you can consider installing.
+
+MongoDB for VS Code::
+MongoDB for VS Code lets you connect to MongoDB and Atlas directly from your VS Code Server environment. Explore databases and collections, inspect schemas, and prototype queries and aggregations using the integrated playgrounds.
+
+link:https://marketplace.visualstudio.com/items?itemName=mongodb.mongodb-vscode[See in Visual Studio Marketplace]
+
+MySQL Shell for VS Code::
+This extension provides a MySQL Shell for VS Code, enhancing your ability to manage MySQL databases within the VS Code Server environment.
+
+link:https://marketplace.visualstudio.com/items?itemName=Oracle.mysql-shell-for-vs-code[See in Visual Studio Marketplace]
+
+SQL Server (MSSQL)::
+Connect to SQL Server with this extension and perform database tasks directly from your VS Code Server environment.
+
+link:https://marketplace.visualstudio.com/items?itemName=ms-mssql.mssql[See in Visual Studio Marketplace]
+
+PostgreSQL::
+This extension facilitates direct interaction with PostgreSQL databases, allowing you to navigate, query, and manage your PostgreSQL instances efficiently.
+
+link:https://marketplace.visualstudio.com/items?itemName=ms-ossdata.vscode-postgresql[See in Visual Studio Marketplace]
+
+SQLTools::
+SQLTools simplifies database connections and enhances data management and query execution, supporting a variety of commonly used databases.
+
+link:https://marketplace.visualstudio.com/items?itemName=mtxr.sqltools[See in Visual Studio Marketplace]
+
+Database Client::
+This extension serves as a database manager for MySQL/MariaDB, PostgreSQL, SQLite, Redis, and ElasticSearch. It provides a unified interface for managing diverse databases within the VS Code Server environment.
+
+link:https://marketplace.visualstudio.com/items?itemName=cweijan.vscode-database-client2[See in Visual Studio Marketplace]
+
+=== Incorporating DBMS clients using a custom image
+
+Custom notebook images are a powerful tool when working with containerized environments, especially when you need specific libraries, OS packages, or applications that are not readily available in base images. Follow these steps to create a custom notebook image with Database Management System (DBMS) clients for MongoDB, MSSQL, and PostgreSQL using Dockerfile.
```

Comment:
The word "readily" could be omitted.

Reply:
Agreed, will remove 'readily'.  How about cutting 'a powerful tool' and using 'useful' instead?

Reply:
sounds good